### PR TITLE
Issue #2048 Moving tests for earlier versions to provisioning various versions feature file

### PR DESCRIPTION
**Pull request by amitkrout**
FIX : #2048

### CODE REVIEW

File: test/integration/features/provision_various_versions.feature [hunk 1]
```diff
@@ -18,6 +18,12 @@ Feature: Provision all major OpenShift versions
       """
       <ocVersion>
       """
+      And "status code" of HTTP request to "/healthz" of OpenShift instance is equal to "200"
+      And "body" of HTTP request to "/healthz" of OpenShift instance contains "ok"
+      And "status code" of HTTP request to "/healthz/ready" of OpenShift instance is equal to "200"
+      And "body" of HTTP request to "/healthz/ready" of OpenShift instance contains "ok"
+      And "status code" of HTTP request to "/console" of OpenShift instance is equal to "200"
+      And "body" of HTTP request to "/console" of OpenShift instance contains "<title>OpenShift Web Console</title>"
```
Comment (budhrg) [inline test/integration/features/provision_various_versions.feature]: @amitkrout , are you deleting `v3.6.0` as it seems redundant to me?

Comment (anjannath) [reply test/integration/features/provision_various_versions.feature]: @budhrg Why do you think it is redundant, is it becaue we are already checking two versions less than v3.6.1, i.e. v1.4.1 and v1.5.1?

Comment (budhrg) [reply test/integration/features/provision_various_versions.feature]: Because, at least I assume v3.6.1 is patch release to v3.6.0

Comment (budhrg) [reply test/integration/features/provision_various_versions.feature]: Also, why we are not testing `1.5.0` it is also listed in `minishift openshift version list`

Comment (amitkrout) [reply test/integration/features/provision_various_versions.feature]: > Because, at least I assume v3.6.1 is patch release to v3.6.0

Make sense

Comment (amitkrout) [reply test/integration/features/provision_various_versions.feature]: Done

File: test/integration/features/provision_various_versions.feature [hunk 2]
```diff
@@ -1,4 +1,4 @@
-@provision-various-versions
+@provision-various-versions @openshift @minishift-only
```
Comment (budhrg) [inline test/integration/features/provision_various_versions.feature]: @amitkrout , could you remove `@minishift-only` here? We can use same testcases with different table in downstream.

Comment (amitkrout) [reply test/integration/features/provision_various_versions.feature]: @budhrg https://github.com/minishift/minishift/pull/2044/files#r170179932

Comment (amitkrout) [reply test/integration/features/provision_various_versions.feature]: @budhrg removed the ```@minishift-only``` tag

**Comment by LalatenduMohanty** [comment]
Updated the commit message and merged the PR 
```
commit 0ef2dec5065f6fb1b141630963bfec4c99a04042 (HEAD -> master, upstream/master, origin/master, origin/HEAD, 2061)
Author: Amit Rout <arout@redhat.com>
Date:   Thu Feb 22 17:24:06 2018 +0530

    Issue #2048 Consolidate tests for provisioning of various versions

    Moving tests for earlier versions to provisioning various versions feature file.
    Now one feature file for cover both test scenarios.
```

**Pull request closed without merging**